src/views/Invite/Cohort.js

```diff
@@ -154,6 +154,8 @@ export default function InviteCohort() {
       getPendingTransactions(point);
       pop();
     } catch (error) {
+      console.log(error);
+      // TODO: error is not used anywhere on the UI
```

Comment:
This is just as a reminder that we need to show any possible errors here in the UI